docs/docusaurus/docs/core/validate_data/checkpoints/_examples/get_checkpoints_by_attributes.py

```diff
@@ -0,0 +1,24 @@
+"""
+This example script demonstrates how to filter the list of Checkpoints in a
+ Data Context on attributes.
```

Comment:
```suggestion
 Data Context on attributes.
```
_on attributes_ is an awkward construction. I'm not sure what it means. Can this be clarified?

Reply:
I'll change it to just "filter the list of Checkpoints available from a Data Context."  Fortunately the new docs format doesn't link directly to these code files, so these docstring is more for internal use and isn't public facing.